src/styles/theme/types.ts

```diff
@@ -27,4 +40,4 @@ export type ICustomThemeProps = {
   pageBackground: string;
 };
 
-export type IThemeProps = IDefaultThemeProps & ICustomThemeProps & IBreakpoints;
+export type IThemeProps = IDefaultThemeProps & ICustomThemeProps & IBreakpoints & IDevices;
```

Comment:
E remover o type IDevices daqui também.

```suggestion
export type IThemeProps = IDefaultThemeProps & ICustomThemeProps & IBreakpoints;
```

Reply:
Ok, vou fazer isso aí 🤙